package.json

```diff
@@ -31,7 +31,7 @@
     }
   },
   "dependencies": {
-    "@actions/core": "^1.6.0",
+    "@actions/core": "^1.8.0",
```

Comment:
Could we update to the latest version `1.8.1`?
https://www.npmjs.com/package/@actions/core/v/1.8.1

Reply:
@joshmgross updated